packages/vscode-extension/src/project/deviceSession.ts

```diff
@@ -355,6 +361,29 @@ export class DeviceSession
         this.emitStateChange();
       }
     });
+    devtools.onEvent("appOrientationChanged", (orientation: AppOrientationType) => {
+      const isLandscape =
+        this.rotation === DeviceRotationType.LandscapeLeft ||
+        this.rotation === DeviceRotationType.LandscapeRight;
+
+      // if the app orientation is equal to "Landscape", it means we do not have enuogh
+      // information on the application site to infer the detailed orientation. 
+      // "Landscape" will only be messaged upon the initialisation - after that the app can infer the orientation
+      // based on our assumptions.
```

Comment:
I feel that this isn't really relevant here.
Here, we don't really care that the "Landscape" orientation will only be passed during initialization, that's a detail of how the code on the application side works -- all we need is the information that "Landscape" is a partial information about app orientation and we need to heuristically pick something ourselves.